src/app/Scenes/MyProfile/Components/UserAccountHeader/UserAccountHeader.tsx

```diff
@@ -0,0 +1,250 @@
+              height={70}
+              width={70}
+              borderRadius={35}
+              backgroundColor={color("mono5")}
+              borderWidth={me?.icon?.url ? undefined : 1}
+              borderColor={me?.icon?.url ? undefined : color("mono10")}
+              alignItems="center"
+              justifyContent="center"
+            >
+              {me?.icon?.url ? (
+                <Flex overflow="hidden" borderRadius={35}>
+                  <Image
+                    src={me.icon.url}
+                    height={70}
+                    width={70}
+                    performResize={false}
+                    testID="profile-image"
+                  />
+                </Flex>
+              ) : (
+                <PersonIcon width={18} height={18} />
+              )}
+              {!!me.isIdentityVerified && (
+                <Flex
+                  width={22}
+                  height={22}
+                  borderRadius={11}
+                  backgroundColor="blue100"
+                  alignItems="center"
+                  justifyContent="center"
+                  position="absolute"
+                  right={0}
+                  bottom={0}
+                  testID="identity-verified-icon"
+                >
+                  <ShieldIcon fill="mono0" />
+                </Flex>
+              )}
+            </Flex>
+
+            {/* Name */}
+            <Text variant="lg-display" color={color("mono100")}>
+              {me.name}
+            </Text>
+
+            {/* Information */}
+            <Flex flexDirection="row" flexWrap="wrap" justifyContent="center">
+              <Join separator={<Spacer x={2} />}>
+                {!!me?.location?.display && (
+                  <Flex flexDirection="row" justifyContent="center" alignItems="center">
+                    <MapPinIcon />
+                    <Text variant="xs" ml={0.5}>
+                      {me.location.display}
+                    </Text>
+                  </Flex>
+                )}
+                {!!me?.profession && (
+                  <Flex flexDirection="row" justifyContent="center" alignItems="center">
+                    <BriefcaseIcon />
+                    <Text variant="xs" ml={0.5}>
+                      {me.profession}
+                    </Text>
+                  </Flex>
+                )}
+                {!!me?.otherRelevantPositions && (
+                  <Flex flexDirection="row" justifyContent="center" alignItems="center">
+                    <InstitutionIcon />
+                    <Text variant="xs" ml={0.5}>
+                      {me.otherRelevantPositions}
+                    </Text>
+                  </Flex>
+                )}
+              </Join>
+            </Flex>
+
+            <MyCollectionPreview me={me} />
+
+            {isProfileComplete ? (
+              <Button
+                variant="outline"
+                size="small"
+                onPress={() => navigate("/my-collection")}
```

Comment:
https://github.com/artsy/eigen/blob/816c8171e80e71806aecf2ee097c03eaef947cc9/docs/routing.md#L15

```suggestion
              <RouterLink hasChildTouchable to="/my-collection">
                <Button
                  variant="outline"
                  size="small"
```

Reply:
I will need some time to adjust to the new way 😄 Thanks!